itest/assets_test.go

```diff
@@ -1938,6 +1969,22 @@ func defaultCoOpCloseBalanceCheck(t *testing.T, local, remote *HarnessNode,
 
 		require.Contains(t, assetIDStrings, assetIDStr)
 
+		// We only check for a proof if an asset of that asset ID was
+		// actually in the close output, which might not always be the
+		// case in grouped asset channels.
+		localAssetIDs := fn.NewSet[string](fn.Map(
+			func(t rfqmsg.JsonAssetTranche) string {
+				return t.AssetID
+			},
+			closedJsonChannel.LocalAssets,
+		)...)
+		if !localAssetIDs.Contains(assetIDStr) {
+			continue
+		}
+
+		t.Logf("Is script key %s (asset ID %s) in list of asset IDs "+
+			"(%v)?", scriptKeyStr, assetIDStr, localAssetIDs)
```

Comment:
maybe we don't need this log message?

Reply:
Ah yes, was left over from debugging. Removed.